app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixUploadImage.kt

```diff
@@ -0,0 +1,115 @@
+              horizontalAlignment = Alignment.CenterHorizontally) {
+                Text(
+                    "Pictures",
+                    style = MaterialTheme.typography.headlineLarge,
+                    color = colorScheme.onBackground,
+                    modifier = Modifier.testTag("picturesText"))
+                Divider(
+                    color = colorScheme.onSecondaryContainer,
+                    thickness = 1.dp,
+                    modifier = Modifier.testTag("divider"))
+
+                // Option to take a photo
+                Row(
+                    modifier =
+                        Modifier.fillMaxWidth()
+                            .clickable {
+                              onTakePhotoClick()
+                              onDismissRequest()
+                            }
+                            .padding(vertical = 8.dp)
+                            .testTag("cameraRow"),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.Start) {
+                      Icon(
+                          painter = painterResource(id = R.drawable.camera),
+                          contentDescription = "Take a photo",
+                          tint = colorScheme.onBackground,
+                          modifier = Modifier.padding(start = 16.dp))
+                      Spacer(modifier = Modifier.width(120.dp))
+                      Text(
+                          "Take a photo",
+                          style = MaterialTheme.typography.headlineSmall,
+                          color = colorScheme.onBackground,
+                          modifier =
+                              Modifier.weight(1f)
+                                  .align(Alignment.CenterVertically)
+                                  .testTag("cameraText"))
+                    }
+
+                Divider(color = colorScheme.onSecondaryContainer, thickness = 1.dp)
+
+                // Option to choose from library
+                Row(
+                    modifier =
+                        Modifier.fillMaxWidth()
+                            .clickable {
+                              onChooseFromLibraryClick()
+                              onDismissRequest()
+                            }
+                            .padding(vertical = 8.dp)
+                            .testTag("libraryRow"),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.Start) {
+                      Icon(
+                          painter = painterResource(id = R.drawable.upload_image),
+                          contentDescription = "Choose from library",
+                          tint = colorScheme.onBackground,
+                          modifier = Modifier.padding(start = 16.dp))
+                      Spacer(modifier = Modifier.width(108.dp))
```

Comment:
It would be better to replace Modifier.width(108.dp) with Modifier.weight() for the spacing, as this absolute width may not display correctly on all phones. Using weights will make the layout more adaptable and ensure consistent spacing across different screen sizes.

Reply:
Right. Thanks for the comment, this will be done in another pr where all absolutes values will be refactored.